src/deploy.sh

```diff
@@ -11,10 +10,9 @@
 #    important to highlight that a VM in the localhost can be treated as a
 #    remote machine.
 #
-# Usually, install modules and update the kernel image requires root
-# permission, with this idea in mind we rely on the `/root` in the remote
-# machine. Additionally, for local deploy, you will be asked to enter your
-# root password.
+# Usually, install modules and update the kernel image requires root permission,
```

Comment:
how about: Usually, **to** install modules and update the kernel image **root permission is required**